plugins/GSdx/stdafx.cpp

```diff
@@ -71,16 +71,54 @@ void vmfree(void* ptr, size_t size)
 	VirtualFree(ptr, 0, MEM_RELEASE);
 }
 
+static HANDLE s_fh = NULL;
+static uint8* s_Next[8];
+
 void* fifo_alloc(size_t size, size_t repeat)
 {
-	// FIXME check linux code
-	return vmalloc(size * repeat, false);
+	ASSERT(s_fh == NULL);
+
+	if (repeat >= countof(s_Next))
+		fprintf(stderr, "Memory mapping overflow (%zu >= %u)\n", repeat, countof(s_Next));
```

Comment:
I think it would be better to return a `nullptr` in case of error. I know that it will still crash later as the `nullptr` isn't handled by the `fifo_alloc` caller.